### DISCUSSION TITLE
A brief history of the HSL

### DISCUSSION THREAD

**Discussion by dpo** [JuliaSmoothOptimizers/HSL.jl#186] [General]
Coincidental with the recent release of HSL_jll.jl, the following report just came out and should interest HSL users:

HSL@60: a brief history of the HSL mathematical software library
J. Scott
https://epubs.stfc.ac.uk/work/54066639

**Comment by amontoison** [comment]
STFC also posted a message on Linkedin today and did a blog:
https://www.scd.stfc.ac.uk/Pages/HSL-60th-Anniversary.aspx

![Screenshot_20230530_114713_LinkedIn](https://github.com/JuliaSmoothOptimizers/HSL.jl/assets/35051714/88d6910e-3d8f-4204-a118-f05057b21e46)
![Screenshot_20230530_114729_LinkedIn](https://github.com/JuliaSmoothOptimizers/HSL.jl/assets/35051714/3c697bc4-b66d-4940-8aa2-1793ef2535ef)
